doc/debian/jitsi-meet/jitsi-meet.example

```diff
@@ -73,6 +73,14 @@ server {
         alias /usr/share/jitsi-meet/libs/external_api.min.js;
     }
 
+    location = /_api_/room-info {
+        proxy_pass http://prosody/room-info?$args;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_set_header Host $http_host;
+        proxy_set_header Connection "";
```

Comment:
Why do you need this header?

Reply:
Copy paste :)

Reply:
@saghul what about the _api_ , any opinions?

Reply:
I've see _foo used, but never _foo_.

I wonder if we could have some generic "_service" or "_internal" tenant-like so we put all internal things there like _service/room-info _service/jaas-something.

Then we can filter easy on nginx.

Reply:
I've see _foo used, but never _foo_.

I wonder if we could have some generic "_service" or "_internal" tenant-like so we put all internal things there like _service/room-info _service/jaas-something.

Then we can filter easy on nginx.

Reply:
> I wonder if we could have some generic "_service" or "_internal" tenant-like so we put all internal things there like _service/room-info _service/jaas-something.

That was the idea of introducing `_api` ... 

I have a problem with the nginx rule as I need to duplicate it to match

`/_api/room-info` and `/_api/room-info/someusertenant`, but I need to discuss that and with Aaron and Scott as I'm not sure how the haproxy mathcing is working ...

Reply:
Ah that looks nice! Maybe I just reviewed an old version!